sdk/src/Services/DynamoDBv2/Custom/DocumentModel/JsonUtils.cs

```diff
@@ -40,11 +37,18 @@ internal static class JsonUtils
         /// <returns></returns>
         public static Document FromJson(string jsonText)
         {
-            var json = JsonMapper.ToObject(jsonText);
-            if (!json.IsObject)
+            var json = JsonDocument.Parse(jsonText, new JsonDocumentOptions
+            {
+                AllowTrailingCommas = true,
```

Comment:
In your description you say 
"A slight change in behavior is the JSON being past into the FromJson is more strictly parsed. Meaning it doesn't allow extra , for the last property in an object. LitJson would allow that. Given that isn't valid JSON this seems an acceptable change for V4."

but here you are allowing trailing commas. Did you set `AllowTrailingCommas` to true on purpose to keep the same behavior as litJson or did you mean to set this to `false`?

Reply:
Oh yeah. I copied the comment from Alex's original PR he must have added setting that property afterwards.